sbin/start-master.sh

```diff
@@ -23,6 +23,26 @@ if [ -z "${SPARK_HOME}" ]; then
   export SPARK_HOME="$(cd "`dirname "$0"`"/..; pwd)"
 fi
 
+# Figure out where Spark is installed
+FWDIR="$(cd "`dirname "$0"`"/..; pwd)"
```

Comment:
This is wrong now, though; like other scripts it should be using `SPARK_HOME` not `FWDIR`